content/build-tool.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Build system
+title: Build System
```

Comment:
I think title should be title case, but headers should be sentence case. That seems consistent with style I see elsewhere, but we could just use sentence case everywhere!